packages/common/src/fees/__tests__/broadcaster-fee-cache.test.ts

```diff
@@ -23,7 +23,7 @@ const initialState: BroadcasterFeeCacheState = {
   forNetwork: {},
 };
 
-const chain = MOCK_CHAIN_ETHEREUM;
+const chain = { id: 42161, type: 0 };
```

Comment:
I'd guess future wise we can move this to constants  cc @jacobmakarsky 

```suggestion
const MOCK_CHAIN_ARBITRUM: Chain = { type: 0, id: 42161 }
const chain = MOCK_CHAIN_ARBITRUM;
```